students.rb

```diff
@@ -4,13 +4,14 @@ class Student < Person
   attr_reader :classroom
 
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission: parent_permission)
-    self.classroom = classroom
+    super(age, name)
+    @classroom = classroom
+    @parent_permission = parent_permission
```

Comment:
- [ ] Well done for attempting to fix the issue of creating a Person (Student or Teacher) which the previous reviewer mentioned 👏. However, there is a new issue which is that the name is now "**unknown**" after saving. This is because you skipped it in the constructor for both `Student` and `Teacher`.
![image](https://user-images.githubusercontent.com/46839548/213864897-b9154ff7-8f81-42af-9288-d370a26e0c8f.png)


**HINT** Add this line in both the `Student` and `Teacher` constructors to fix this error 👇 
```ruby
@name = name
```

Reply:
thanks alot